views/js/review/plugins/navigation/review-panel/scss/panel.scss

```diff
@@ -227,7 +228,6 @@ $incorrectColor: $error;
 
     .review-panel-content {
         flex: 0 1 auto;
-        overflow-y: auto;
```

Comment:
Why is this removed? The component is now broken when the amount of elements make the panel bigger than the available space.

Reply:
restored